src/main/java/core/basesyntax/AreaCalculation.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface AreaCalculation {
+    double getArea();
+
+    void draw();
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-put-all-behaviour-into-a-single-interface-if-the-methods-are-conceptually-different-from-each-other

Reply:
create one more Interface for methid drow